transaction-ballerina/tests/transactional_expr.bal

```diff
@@ -17,6 +17,7 @@
 import ballerina/test;
 
 @test:Config {
+    enable:false
```

Comment:
This is due to bug in current transaction modules as after retry transaction, strand does not get updated for set non transactional. Previously each test run in separate strand so tests were passed. These tests are individually passing.